hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -346,6 +347,18 @@ public int getHBaseIndexDesiredPutsTime() {
     return Integer.valueOf(props.getProperty(HoodieHBaseIndexConfig.HOODIE_INDEX_DESIRED_PUTS_TIME_IN_SECS));
   }
 
+  public boolean enableAutoTuneBloomFilter() {
```

Comment:
the filter type is not a boolean right?

Reply:
this method is not required. sorry forgot to remove